routes/userroutes.js

```diff
@@ -0,0 +1,14 @@
+////THIS FILE CONTAINS ALL USER ROUTES
+const express = require("express");
+const httpStatusCodes = require("http-status-codes");
+const userController = require("../controller/usercontroller");
+
+const router = express.Router();
+const user = new userController();
+
+router.get("/", user.getAllUsers);
+router.post("/", user.createUser);
+router.post("/:name/:age/:email", user.addUser);
```

Comment:
You should use the `body` here to send data through a `POST` request and then receive it through `req.body`.